app/konnect/org-management/okta-idp.md

```diff
@@ -157,8 +157,21 @@ values are present.
 
     Refer back to your Okta application to fill in the following fields.
 
-1. Copy the **Issuer URI** from your Okta custom authorization server, then paste it into
-the **Issuer URI** field in {{site.konnect_short_name}}. <!-- found in general settings in Okta app -->
+1. Locate your issuer URI in Okta.
+    1. Go to **Security** > **API**.
+    1. Copy the issuer URI for your authorization server. It should look
+    something like this:
+
+        ```
+        https://example-account.okta.com/oauth2/default
```

Comment:
```suggestion
        https://EXAMPLE_ACCOUNT.okta.com/oauth2/DEFAULT
```

Reply:
okay feel free to ignore this because I think it looks ugly but we have to figure out what to do in these cases

Reply:
What about this? 
```suggestion
        https://{example-account}.okta.com/oauth2/default
```

Reply:
But then we're back to the issue of using `{ }` where it might be confusing. 

I think what we're running up against here is fake URL vs placeholder. Even the Google style guide makes that distinction:

https://developers.google.com/style/examples

Gitlab has a good section as well (much smaller): https://docs.gitlab.com/ee/development/documentation/styleguide/#fake-urls

In URLs especially, that tends to be the prevailing style.

Reply:
makes sense to me. I definitely wouldn't change `/default` to `/DEFAULT` as a lot of times, that is in fact the correct value if they only have a single app set up